autoapi/backends.py

```diff
@@ -25,3 +25,35 @@
     "go": GoSphinxMapper,
     "javascript": JavaScriptSphinxMapper,
 }
+
+
+#: describes backend requirements in form
+#: {'backend name': (('1st package name in pypi', '1st package import name'), ...)}
+backend_requirements = {
+    "python": (),
+    "javascript": (),
+    "go": (("sphinxcontrib-golangdomain", "sphinxcontrib.golangdomain"),),
+    "dotnet": (("sphinxcontrib-dotnetdomain", "sphinxcontrib.dotnetdomain"),),
+}  # type: (Dict[str, Sequence[Tuple[str, str]]])
+
+
+def _get_available_backends():
```

Comment:
Do you think it will simplify this if we check for the module in `sys.modules` instead (probably in `autoapi/extension.py` rather than here)? A user should have specified the domain extension in the `extensions` list in their `conf.py` already, so we can check that the extension is loaded.

Reply:
That's is good point, will do in this way